src/components/tree/tree.js

```diff
@@ -110,18 +115,17 @@ class Tree extends React.Component {
   }
 
   getStyles = () => {
-    const activeResetTreeButton = this.props.tree.idxOfInViewRootNode !== 0 ||
-      this.props.treeToo.idxOfInViewRootNode !== 0;
-
     const filteredTree = !!this.props.tree.idxOfFilteredRoot &&
       this.props.tree.idxOfInViewRootNode !== this.props.tree.idxOfFilteredRoot;
     const filteredTreeToo = !!this.props.treeToo.idxOfFilteredRoot &&
       this.props.treeToo.idxOfInViewRootNode !== this.props.treeToo.idxOfFilteredRoot;
     const activeZoomButton = filteredTree || filteredTreeToo;
 
-    const treeIsZoomed = this.props.tree.idxOfInViewRootNode !== 0 ||
+    const anyTreeZoomed = this.props.tree.idxOfInViewRootNode !== 0 ||
       this.props.treeToo.idxOfInViewRootNode !== 0;
 
+    const activeResetTreeButton = anyTreeZoomed;
```

Comment:
if you're gonna make these two variables the same thing why not just go the one extra step and rename `treeIsZoomed` directly to `activeResetTreeButton`, and then update the ternaries below to also use `activeResetTreeButton`? Seems less confusing in the longer run.

Reply:
Good point, done: 4d8a2e04

I chose to go the other way around and drop `activeResetTreeButton` in favor of `anyTreeZoomed` to convey the condition rather than end state.